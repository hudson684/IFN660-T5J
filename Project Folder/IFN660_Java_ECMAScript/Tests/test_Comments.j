test1
/*comment*/

test2
/* comment with space */

test3
/* Multiline 
   comment
 */

test4
/* comment */
foo
/* comment */
test4

test5
/*
 * Multiline comment
 */
 /* Second comment */

test6
/****
 * Common style
 ****/
test6
/*
 * Comment
 */
test6

test7
/*****
 * Another comment style
 */

test8
// singleline comment