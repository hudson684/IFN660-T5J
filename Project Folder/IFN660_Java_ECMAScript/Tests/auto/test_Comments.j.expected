IDENT (test1)
IDENT (test2)
IDENT (test3)
IDENT (test4)
IDENT (foo)
IDENT (test4)
IDENT (test5)
IDENT (test6)
IDENT (test6)
IDENT (test6)
IDENT (test7)
IDENT (test8)
EOF